pkgs/development/compilers/purescript/purescript/default.nix

```
@@ -0,0 +1,53 @@
+{ pkgs ? import <nixpkgs> {} }:
```

In nixpkgs, we never want to re-import nixpkgs. Instead, we want to receive the dependent packages as parameters. For example:

```suggestion
{ stdenv, fetchurl, zlib, gmp, ncurses5 }:
```

and then subsequent references drop the `pkgs.` prefix.